### PR TITLE
fix(TPRUN-5024) : remove camel-script-jruby feature

### PR DESCRIPTION
**Pull request by f2par0**
🏁 **Context (Jira, design, ...)**
- https://jira.talendforge.org/browse/TPRUN-5024
- 
🔍 **What is the problem this PR is trying to solve?**
jruby embeds snakeyaml libraries with CVEs

🚀 **What is the chosen solution to this problem?**
Empty the feature (removal could cause hangs during update)

🎾 **Impacts**

- [ ] **This PR introduces a breaking change**

**Dear contributor, please check if the PR fulfills these requirements**

- [x] The PR commit message follows our [guidelines](https://github.com/talend/tools/blob/master/tools-root-github/CONTRIBUTING.md)
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)
- [ ] Related design / discussions / pages (not in jira), if any, are all linked or available in the PR

### CODE REVIEW

File: platforms/karaf/features/src/main/resources/features.xml [hunk 1]
```diff
@@ -2089,8 +2089,7 @@
     <bundle>mvn:org.apache.camel/camel-schematron/${upstream.version}</bundle>
   </feature>
    <feature name='camel-script-jruby' version='${upstream.version}' resolver='(obr)' start-level='50'>
-    <feature version='${upstream.version}'>camel-script</feature>
-    <bundle>mvn:org.jruby/jruby-complete/${jruby.tesb.version}</bundle>
+    <!-- removed for security reasons -->
```
Comment (andreasmattes) [inline platforms/karaf/features/src/main/resources/features.xml:2092]: I suggest to add to the comment: "CVE-2022-25857 (high, snakeyaml included in jruby) and further medium CVEs". This clarifies better while this feature has been cleared.

**Review by andreasmattes — CHANGES_REQUESTED** [review]
Addition of the CVE to be considered, the rest looks good.

File: platforms/karaf/features/src/main/resources/features.xml (hunk 1 above)
Comment (f2par0) [reply platforms/karaf/features/src/main/resources/features.xml:2092]: Ok, done

Comment (andreasmattes) [reply platforms/karaf/features/src/main/resources/features.xml:2092]: CVE reference is added.

**Pull request merged**